utilities/fuzzer.py

```diff
@@ -488,17 +516,27 @@ def generate_tests(self):
 
         returns (filename, object)
         """
+        ##### Statetest Template init
+        from evmlab.tools.statetests.templates import statetest
 
         # We'll offload test generation to another thread
         q = queue.Queue(maxsize = 20)
         def createATest():
             counter = 0
             while True:
-                test_obj = self.statetest_template.fill()
+                statetest_template = statetest.StateTestTemplate(nonce="0x1d",
```

Comment:
better fix in #116 
https://github.com/ethereum/evmlab/pull/116/files#diff-69770162e2ecacf29d6bf49c29b8eef2R518